example/vectorAdd/src/vectorAdd.cpp

```diff
@@ -108,7 +109,8 @@ auto main() -> int
 
     // Get the host device for allocating memory on the host.
     using DevHost = alpaka::DevCpu;
-    auto const devHost = alpaka::getDevByIdx<DevHost>(0u);
+    auto const platformHost = alpaka::PltfCpu{};
+    DevHost const devHost = alpaka::getDevByIdx(platformHost, 0);
```

Comment:
Same for the host platform, it has to be instantiated.

I wonder whether we should have an alias `HostPlatform` and `HostDevice` inside alpaka. Or potentially even a global object for the host platform. The latter however would not allow us again to cleanly shutdown everything object in alpaka.